### PR TITLE
Add Matterport provider

### PR DESCRIPTION
**Pull request by papes1ns**
Supports rich interactive Matterport embeds without the need for relying on the fallback aggregators.

Matterport lets you create and embed 3D models of buildings and spaces, e.g. https://my.matterport.com/show/?m=FmDYedjofjo. Credit to @haeric for finding the [Matterport oEmbed provider URL](https://github.com/iamcal/oembed/pull/338/files).

### CODE REVIEW

File: lib/oembed/providers/builtin_providers.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
 require 'oembed/providers/facebook_video'
 require 'oembed/providers/instagram'
 require 'oembed/providers/tiktok'
+require 'oembed/providers/matterport'
```
Comment (metavida) [inline lib/oembed/providers/builtin_providers.rb:9]: **Nit:** I have a slight preference for keeping these lines in alphabetical order (unless there's a specific reason not to)

```suggestion
require 'oembed/providers/matterport'
require 'oembed/providers/tiktok'
```

**Review by metavida — APPROVED** [review]
Thanks for contributing a new provider! I'm not exactly sure when I'll have time to release a new version of the gem, but I've got it on my TODO list for when I do manage to find the free time.

File: lib/oembed/providers/builtin_providers.rb (hunk 1 above)
Comment (papes1ns) [reply lib/oembed/providers/builtin_providers.rb]: I agree with that - fixed 👍

**Comment by papes1ns** [comment]
Appreciate the feedback!

Really enjoy using this gem. Decided to contribute by officially adding Matterport as a provider, aiming to simplify integration for future devs and spare them the hassle of hunting down the right docs. They were not trivial to find :)

**Comment by metavida** [comment]
FYI, I released the Materport provider with [0.17.0](https://rubygems.org/gems/ruby-oembed/versions/0.17.0) back on February 25. Thanks again for the contribution & so glad you like the gem!

**Pull request merged**